packages/core/src/accordion/AccordionHeader.css

```diff
@@ -34,9 +30,37 @@
 }
 
 .saltAccordionHeader-icon {
-  transition: transform var(--salt-duration-instant) cubic-bezier(0.4, 0, 0.2, 1);
+  transition: transform var(--salt-duration-perceptible) ease-in-out;
 }
 
 .saltAccordionHeader[aria-expanded="true"] > .saltAccordionHeader-icon {
   transform: rotate(90deg);
 }
+
+.saltAccordionHeader-error {
+  background: var(--salt-status-error-background);
+}
+
+.saltAccordionHeader-error:focus-visible {
+  outline-color: var(--salt-status-error-foreground);
+}
+
+.saltAccordionHeader-warning {
+  background: var(--salt-status-warning-background);
+}
+
+.saltAccordionHeader-warning:focus-visible {
+  outline-color: var(--salt-status-warning-foreground);
+}
+
+.saltAccordionHeader-success {
+  background: var(--salt-status-success-background);
+}
+
+.saltAccordionHeader-success:focus-visible {
+  outline-color: var(--salt-status-error-foreground);
```

Comment:
```suggestion
  outline-color: var(--salt-status-error-borderColor);
```
for these outlines?

Reply:
Probably, I was copying what has been done in other components. We'll probably have to update those since this will introduce inconsistency